lib/psbt.ex

```diff
@@ -52,9 +59,22 @@ defmodule Bitcoinex.PSBT do
       global <> inputs <> outputs
   end
 
+  @doc """
+    to_file writes a PSBT to file as binary.
+  """
+  @spec to_file(%Bitcoinex.PSBT{}, String.t()) :: atom
```

Comment:
File.close return` :ok | {:error, posix() | :badarg | :terminated}`. Since we are already only pattern match only ok case for File.open. I will add `!` as suffix like `to_file!` to signal this function will raise exception for exception case and change the result type `:ok` so sth like that

```
@spec to_file!(%Bitcoinex.PSBT{}, String.t()) :: :ok
def to_file(packet, filename) do
   ....
   :ok = File.close(file)
  :ok
end
```